apache-apisix/metadata.csv

```diff
@@ -0,0 +1,7 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name

```

Comment:
Hi @bisakhmondal, could you please help to check the metric type according to https://docs.datadoghq.com/metrics/types/?tab=count

Reply:
Hi! Both Histograms and timers sent to dogstatsd are combinations of rate and gauge. I guess it's better to add 2 entries for those metric names.

Reply:
@bisakhmondal Could you please give suggestions here?

![image](https://user-images.githubusercontent.com/2106987/145337611-09df3272-d8ca-4e8f-abd8-7930ee6c03e9.png)

So I could use Commit Directly 😄

Reply:
Ahh cool. I can do it.

Reply:
done